controllers/record_match_run.go

```diff
@@ -96,7 +103,7 @@ func CreateRecordMatchRunHandler(provider func() *mgo.Database) gin.HandlerFunc
 				"server endpoint": svrEndpoint,
 				"reqBody":         string(reqBody[:]),
 				"message":         reqMatchRequest.Message,
-				"request":         reqMatchRequest}).Info("About to submit request")
+				"request":         string(reqBody)}).Info("About to submit request")
```

Comment:
Very minor nit, but it appears that we are logging reqBody twice. Once in "reqBody" and once in "request".